src/App.jsx

```diff
@@ -1,3 +1,42 @@
+import Auth from './views/Auth/Auth';
+import { BrowserRouter, Redirect, Route, Switch } from 'react-router-dom';
+import { PrivateRoute } from './components/PrivateRoute';
+import BulletBoard from './views/BulletBoard/BulletBoard';
+import { UserProvider } from './context/context';
+import Details from './views/Details/Details';
+import Header from './components/Header/Header';
+import AddPost from './components/AddPost/AddPost';
+import Edit from './components/Edit/Edit';
+
 export default function App() {
-  return <h1>Hello World</h1>;
+  return (
+    <div className="App">
+      <UserProvider>
+        <BrowserRouter>
+          <Header />
+          <Switch>
+            <Route exact path="/login">
+              <Auth />
+            </Route>
+
+            <Route exact path="/">
+              <BulletBoard />
+            </Route>
```

Comment:
Let's make this a private route so that non authenticated users cannot view any data.